src/main/java/org/csanchez/jenkins/plugins/kubernetes/pod/retention/PodRetention.java

```diff
@@ -45,7 +45,7 @@ public static PodRetention getPodTemplateDefault() {
 
     @Override
     public String toString() {
-        return getDescriptor().getDisplayName();
+        return getClass().getSimpleName();
```

Comment:
IIUC this is part of an ID stored on disk. Is this change going to be a breaking one on plugin update? IOW will the plugin be able to handle what's already stored after this change?

Reply:
I can probably make it compatible.

Reply:
f85374a